ee/pages/api/integrations/stripepayment/webhook.ts

```diff
@@ -26,16 +34,63 @@ export const config = {
 
 async function handlePaymentSuccess(event: Stripe.Event) {
   const paymentIntent = event.data.object as Stripe.PaymentIntent;
-  const isExistingPayment = await prisma.payment.findUnique({
+  const existingPaymentData = await prisma.payment.findUnique({
     where: { externalId: paymentIntent.id },
-    select: { externalId: true },
+    select: {
+      externalId: true,
+      booking: {
+        select: {
+          user: { select: { thetisId: true, username: true } },
+          startTime: true,
+          eventType: { select: { slug: true } },
+          attendees: {
+            select: {
+              name: true,
+              email: true,
+            },
+          },
+        },
+      },
+    },
   });
-  if (!isExistingPayment) {
+  if (!existingPaymentData) {
     throw new HttpCode({
       statusCode: 202,
       message: `Stripe Webhook event received for payment not made in Vulcan.`,
     });
   }
+
+  if (existingPaymentData.booking?.user?.thetisId) {
+    const eventUrl = `${process.env.BASE_URL}/${existingPaymentData.booking?.user?.username}/${existingPaymentData.booking?.eventType?.slug}`;
```

Comment:
Business logic issue that I just discovered - Marketing actually needs the calendar URL for the full priced event, can you hardcode `eventUrl` to be  `${process.env.BASE_URL}/${existingPaymentData.booking?.user?.username}/30min`